samples/snippets/csharp/VS_Snippets_CLR_System/system.Console.WriteLine/CS/WriteLine6.cs

```diff
@@ -8,8 +8,8 @@ public static void Main()
       Random rnd = new Random();
       // Generate five random Boolean values.
       for (int ctr = 1; ctr <= 5; ctr++) {
-         Boolean bln = Convert.ToBoolean(rnd.Next(0, 2));
-         Console.WriteLine("True or False: {0}", bln);
+         bool bln = rnd.Next(0, 2) == 1;

```

Comment:
I could've done it with `is 1`, but I assumed that it would raise additional questions among the newcomers

Reply:
I'd leave this one as is.